packages/airnode-node/src/evm/handlers/initialize-provider.ts

```diff
@@ -109,14 +110,12 @@ export async function initializeProvider(
   );
   logger.logPending(verifyRrpTriggersLogs, baseLogOptions);
 
-  const [verifyWithdrawalLogs, verifiedWithdrawals] = verification.verifySponsorWallets(
-    state4.requests.withdrawals,
-    state4.masterHDNode
-  );
-  logger.logPending(verifyWithdrawalLogs, baseLogOptions);
-
   const state5 = state.update(state4, {
-    requests: { ...state3.requests, apiCalls: verifiedApiCallsForRrpTriggers, withdrawals: verifiedWithdrawals },
+    requests: {
+      ...state3.requests,
+      apiCalls: verifiedApiCallsForRrpTriggers,
+      withdrawals: state4.requests.withdrawals,
```

Comment:
Do we really need to override withdrawals since they haven't changed from state4?